debian/copyright

```diff
@@ -1,5 +1,29 @@
-Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: revpi-wallpaper
+Upstream-Contact: KUNBUS GmbH <support@kunbus.com>
+Source: https://github.com/RevolutionPi/revpi-wallpaper
 
-Files: *
-Copyright: 2016-2021 Kunbus GmbH
-License: proprietary
+Files:     *
+Copyright: 2016-2023 Kunbus GmbH
+License:   GPL-2+
```

Comment:
Is a GPL the correct license for an image? Wouldn't a creative commons license more appropriate? This is a more general question for this kind of content.

Reply:
Makes sense. CC-BY-SA-4.0 is a good fit (even better as CC-BY-SA-3.0 because of the attribution & removal part)